src/components/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,79 @@
+import { FormatDate, NextLinkWrapper } from 'components';
+import { FaSearch } from 'react-icons/fa';
+import styles from './SearchResults.module.scss';
+
+/**
+ * A component that mimics a search result in a loading state for better
+ * perceived performance.
+ *
+ * @returns {React.ReactElement} The LoadingSearchResults component.
+ */
+export function LoadingSearchResult() {
```

Comment:
You should extract this Component on its own file next to the `SearchResults.js` with the name LoadingSearchResult.js.

You can take a look at the Component Design Standards that I've compiled to review this Monday:

https://wpengine.atlassian.net/wiki/spaces/ATLAS/pages/3075211475/Atlas+Blueprints+Component+Design+Standards